### PR TITLE
added nuspec

### PR DESCRIPTION
**Pull request by acesnik**

### CODE REVIEW

File: TopDownProteomics.nuspec [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>TopDownSDK</id>
+    <version>0.0.1</version>
+    <title>TopDownSDK</title>
+    <authors>Center for Top-Down Proteomics</authors>
```
Comment (rfellers) [inline TopDownProteomics.nuspec]: Should be "Consortium for Top-Down Proteomics"

File: TopDownProteomics.nuspec [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    <owners>Center for Top-Down Proteomics</owners>
```
Comment (rfellers) [inline TopDownProteomics.nuspec]: Should be "Consortium for Top-Down Proteomics"

Comment (acesnik) [reply TopDownProteomics.nuspec]: Oops, thanks! Good catch.

**Comment by acesnik** [comment]
I changed Center to Consortium, and I also titled the NuGet package "TopDownProteomics" instead of "TopDownSDK." Sound good to you?

**Comment by rfellers** [comment]
TopDownProteomics is better, although personally TopDownSDK is prettier

**Pull request merged**